packages/@ourworldindata/explorer/src/Explorer.tsx

```diff
@@ -452,6 +456,12 @@ export class Explorer
         newGrapherParams.tab =
             this.grapher.mapGrapherTabToQueryParam(previousTab)
 
+        // reset map state if switching to a chart
+        if (newGrapherParams.tab !== GRAPHER_TAB_QUERY_PARAMS.map) {
+            newGrapherParams.globe = "0"
+            newGrapherParams.mapSelect = ""
+        }
+
```

Comment:
Debatable (same for Mdims)

Reply:
Yeah, I think that makes sense and is a decent choice.